charts/datadog/templates/_daemonset-volumes-linux.yaml

```diff
@@ -9,10 +9,13 @@
 - hostPath:
     path: /sys/fs/cgroup
   name: cgroups
+{{- if or .Values.datadog.systemProbe.osReleasePath .Values.datadog.osReleasePath }}
 - hostPath:
     path: {{ .Values.datadog.systemProbe.osReleasePath | default .Values.datadog.osReleasePath }}
   name: os-release-file
+{{- end }}
 {{- if eq (include "should-enable-system-probe" .) "true" }}
+{{- if .Values.datadog.systemProbe.enableDefaultOsReleasePaths }}
```

Comment:
with `enableDefaultOsReleasePaths: true` by default, if the user specifies `datadog.systemProbe.osReleasePath: /etc/lsb-release` the hostPath `os-release-file` will conflict with `etc-lsb-release`?

Reply:
no it will create 2 volumes, but on will not be use by a volumeMount